rasa/model_training.py

```diff
@@ -124,6 +124,28 @@ def _check_unresolved_slots(domain: Domain, stories: StoryGraph) -> None:
     return None
 
 
+def _check_restricted_slots(domain: Domain) -> None:
+    """Checks if there are any restricted slots.
+
+    Args:
+        domain: The domain.
+
+    Raises:
+        Warn user if there are any restricted slots.
+
+    Returns:
+        `None` if there are no restricted slots.
+    """
+    restricted_slot_names = [rasa.shared.constants.CONTEXT]
```

Comment:
suggestion: import the variable using `from rasa.shared.constants import CONTEXT` and then you can use it in the code directly. That's usually how we use it in the code

Reply:
thanks for the review @twerkmeister